src/popcorn-list-movies/test/PopcornListMovies.test.js

```diff
@@ -7,32 +7,36 @@ const scopedFixture = (template) => fixture(template, { scopedElements });
 
 const movies = [
   {
-    "title": "Batman Begins",
-    "id": "tt0372784",
-    "poster": "/.web-test-runner/assets/img/batman.jpg"
-    },
-    {
-    "title": "Batman v Superman: Dawn of Justice",
-    "id": "tt2975590",
-    "poster": "/.web-test-runner/assets/img/uniquePoster.jpg"
-    },
-]
+    title: "Batman Begins",
+    id: "tt0372784",
+    poster: "/.web-test-runner/assets/img/batman.jpg",
+  },
+  {
+    title: "Batman v Superman: Dawn of Justice",
+    id: "tt2975590",
+    poster: "/.web-test-runner/assets/img/uniquePoster.jpg",
+  },
+  {
+    title: "Spiderman",
+    id: "tt0100669",
+    poster: "N/A",

```

Comment:
You should have another movie object without "poster" property to test that scenario as well.